cli/index.ts

```diff
@@ -159,11 +159,17 @@ async function serve(port: number, pathToSpec: string, options: Options = {}) {
   server.listen(port, () => console.log(`Server started: http://127.0.0.1:${port}`));
 
   if (options.watch && existsSync(pathToSpec)) {
+    const pathToSpecDirectory = dirname(pathToSpec);
+    const watchOptions = {
+      recursive: true
+    };
+
     watch(
-      pathToSpec,
+      pathToSpecDirectory,
+      watchOptions,
       debounce(async (event, filename) => {
-        if (event === 'change' || (event === 'rename' && existsSync(filename))) {
-          console.log(`${pathToSpec} changed, updating docs`);
+        if (event === 'change' || event === 'rename') {
```

Comment:
I am using IntelliJ and it has a feature called 'safe write'. It first renames the original file to `<original_name>___jb_old___`, writes the new content to the original file and then deletes the copy. I think this happens so fast that `existsSync(filename)` always returns `false`. Therefore I removed the check.

Reply:
I am not sure about this change. I will test it locally with my editor on weekends and then will merge it!
thanks!